### PR TITLE
2b) add cpp and rust tests on ci 

### PR DESCRIPTION
**Pull request by crop2000**
based on #1068 
a lot of work was caused by #1075

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -13,11 +13,29 @@ jobs:
...
+         mkdir -p faust/dsp/
+         cp compiler/generator/libfaust.h faust/dsp/
+
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+
+    - name: Run impulse tests (C++)
+      run: cd tests/impulse-tests && make travis
```
Comment (bluenote10) [inline .github/workflows/ubuntu.yml:38]: Strange that the target is called "travis" (tests don't run on Travis CI, no?). But of course that's somewhat independent of this PR, if it happens to be called like this already.

Comment (crop2000) [reply .github/workflows/ubuntu.yml:38]: It is a subset of the cpp tests that finishes in below 1h

**Comment by crop2000** [comment]
@sletz what do you think about this? 
To run the tests takes about 80min.

**Comment by sletz** [comment]
This is too long, I suggest modifying the `make travis` target to only keep the first line.

**Comment by crop2000** [comment]
@sletz is the travis target used elsewhere? if not i would rename it to github_action and describe a meaning full subset. maybe it makes sense to have a detailed check that is not part of the default actions, but can be activated to ensure a thorough check for more complex changes.

**Comment by crop2000** [comment]
@sletz down to ~20m for everything ~5min build faust, ~15min c++ and rust

**Pull request merged**